tests/testing_data/all/chat_client_config.yml

```diff
@@ -0,0 +1,32 @@
+config:
```

Comment:
change to the real chat client config as seen in template/chat_client_config.yml